src/styles/blocks/catalog.scss

```diff
@@ -1,3 +1,5 @@
+$hover-color: #34568b;
```

Comment:
usually we give more generic names for our css-variables. If you gonna need to use this color not on hover - will the name be a good choice?